components/script/dom/htmlmediaelement.rs

```diff
@@ -240,6 +242,7 @@ impl HTMLMediaElement {
             duration: Cell::new(f64::NAN),
             playback_position: Cell::new(0.),
             default_playback_start_position: Cell::new(0.),
+            volume: Cell::new(1.0),
```

Comment:
I initialized it with 1.0 following the specs https://html.spec.whatwg.org/multipage/media.html#dom-media-volume that says that "A media element has a playback volume, which is a fraction in the range 0.0 (silent) to 1.0 (loudest). Initially, the volume should be 1.0, but user agents may remember the last set value across sessions, on a per-site basis or otherwise, so the volume may start at other values."